app/models/compute_resource.rb

```diff
@@ -73,6 +73,10 @@ def self.providers
     end
   end
 
+  def self.providers_requiring_url
+    "Libvirt, oVirt, OpenStack and Rackspace"
```

Comment:
I believe this ought to be a translation friendly string , `_('Libvirt, oVirt, OpenStack and Rackspace')`